rebar.config.script

```diff
@@ -1,7 +1,17 @@
 %% -*- erlang -*-
-case os:getenv("TRAVIS") of
-    "true" ->
-	JobId = os:getenv("TRAVIS_JOB_ID"),
-	lists:keystore(coveralls_service_job_id, 1, CONFIG, {coveralls_service_job_id, JobId});
-    _ -> CONFIG
+case {os:getenv("GITHUB_ACTIONS"), os:getenv("GITHUB_TOKEN")} of
+    {"true", Token} when is_list(Token) ->
+        CONFIG1 = [{coveralls_repo_token, Token},
+        {coveralls_service_job_id, os:getenv("GITHUB_RUN_ID")},
+        {coveralls_commit_sha, os:getenv("GITHUB_SHA")},
+        {coveralls_flag_name, os:getenv("COVERALLS_FLAG_NAME")} | CONFIG],
+        case os:getenv("GITHUB_EVENT_NAME") =:= "pull_request"
+            andalso string:tokens(os:getenv("GITHUB_REF"), "/") of
+            [_, "pull", PRNO, _] ->
+                [{coveralls_service_pull_request, PRNO} | CONFIG1];
+            _ ->
+                CONFIG1
+        end;
+    _ ->
+        CONFIG
```

Comment:
indention is wrong

Reply:
8 white spaces, what spaces size is expected?

Reply:
tabs, 8 spaces should be one tab. Also the alignment of the `CONFIG1` list is wrong.

Reply:
Created separate PR https://github.com/travelping/capwap/pull/38/files